print-workspace-status.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat <<EOF
+E2E_NAMESPACE "${BUILD_ID:-0}"
```

Comment:
Consider:
a) `$USER-$BUILD_ID`
b) only appending `-$BUILD_ID` when $USER == prow

(you can do this kind of bash logic above the `cat` line)

Also, would this be a good candidate for `STABLE_E2E_NAMESPACE`? https://docs.bazel.build/versions/master/user-manual.html#flag--workspace_status_command